README.md

```diff
@@ -703,11 +698,22 @@ startup’s growth and how that affects who owns the stock.
   For this reason, the equity team members get is higher in the earlier stages (starting
   with founders) and increasingly lower as a company matures.
   (See the picture above.)
+- 🄳 At some point early on, generally before the first employees are hired, stock will be
+  allocated to an employee stock
+  [**option pool**](http://www.investopedia.com/terms/o/option-pool.asp). A typical size for this is
+  20% of the stock of the company, but it can be 10%, 15%, or other sizes.
+  Once the pool is established, then the company’s board of directors grants pieces of it to
+  employees as they join the company.
+- The whole pool may never be fully used, or it may be increased later.
+  The size of the pool is determined by
+  [complex factors between founders and investors](http://venturehacks.com/articles/option-pool-shuffle).
+  It’s worth employees understanding that a small pool can be a good thing in that it
+  reflects the company preserving ownership in negotiations with investors.
 
 ### Counting shares
 
-There are important subtleties in how outstanding shares are counted.
-We mention the key ways you might hear of or read about in documents:
+There are subtleties in how [outstanding shares](#stock-and-shares) are counted.
```

Comment:
Consider moving defining occurrence of outstanding shares to the top of this section.

Reply:
Considered that from your PR but don't think I agree, as these are subtleties in _how_ outstanding shares are counted (which can be skipped unless needed) and outstanding shares is the fundamental concept (which is essential to understand).